source/draw/Color.ooc

```diff
@@ -24,6 +24,9 @@ ColorMonochrome: cover {
 	toRgba: func -> ColorRgba { this toYuv() toRgba() }
 	equals: func (other: This) -> Bool { this y == other y }
 	mix: static func (a, b: This, ratio: Float) -> This { This new(Float mix(a y, b y, ratio) as Byte) }
+	biMix: static func (delta: FloatPoint2D, bottomLeft, topLeft, bottomRight, topRight: This) -> This {
```

Comment:
Also want to point out that the order of arguments are reversed compared to the old `mix`, and maybe `delta` should be renamed `ratio`.